### DISCUSSION TITLE
Zk sync testnet not deploying contract

### DISCUSSION THREAD

**Discussion by dannie-jeller** [Cyfrin/foundry-full-course-cu#2134] [Q&A]
Hi guys. i tried dep;oying a contract on the zk sync sepolia testnet, but it keeps rolling. It just doesnt deploy. It just shows deploying contract but never actually deploys

**Comment by EngrPips** [comment]
Hello @dannie-jeller That seems like an internet connection issue. Can you confirm that your internet connection is stable?
